Services/Services/RecruitmentService.cs

```diff
@@ -168,6 +173,23 @@ public RecruitmentListing GetRecruitments(Paging paging, SortOrder sortOrder, Re
             IQueryable<Recruitment> recruitments = _recruitmentRepo.GetAll();
             recruitments = recruitments.Where(r => !r.DeletedDate.HasValue);
 
+            if (recruitmentFiltringDTO.ShowPrivate == false)
+            {
+                recruitments = recruitments.Where(s => s.IsPublic == true);
+            }
+            if (recruitmentFiltringDTO.ShowOpen == false && recruitmentFiltringDTO.ShowClosed == true)
+            {
+                recruitments = recruitments.Where(s => s.EndedDate.HasValue || s.EndingDate < DateTime.Now);
+            }
+            if (recruitmentFiltringDTO.ShowOpen == true && recruitmentFiltringDTO.ShowClosed == false)
+            {
+                recruitments = recruitments.Where(s => !s.EndedDate.HasValue &&
+                s.BeginningDate < DateTime.Now && s.EndingDate > DateTime.Now);
+            }
+            if(recruitmentFiltringDTO.ShowOpen==false && recruitmentFiltringDTO.ShowClosed==false)
+            {
+                recruitments = Enumerable.Empty<Recruitment>().AsQueryable();
```

Comment:
You can return recruitments at this point

Reply:
![Zrzut ekranu (806)](https://user-images.githubusercontent.com/107063119/179691059-fe7fa4d2-32dc-431e-a1e3-dfad0f300394.png)

Musiało by to tak wyglądać, bo muszę zwrócić obiekt klasy RecruitmentListing.